.github/workflows/matrix.json

```diff
@@ -5,7 +5,7 @@
         {
           "php": "8.2",
           "symfony": "^6.4",
-          "api-platform": "^3.1"
+          "api-platform": "^3.3"
```

Comment:
```suggestion
          "api-platform": "3.3.*"
```
🤔

Reply:
The current form also reflects the composer, so in theory, if 3.4 comes out, we would support it and test it straight away 🤔

Reply:
Sounds good 👍